debt-token-interface/src/lib.rs

```diff
@@ -0,0 +1,30 @@
+#![deny(warnings)]
+#![no_std]
+
+use soroban_sdk::{contractclient, contractspecfn, Address, Bytes, Env};
+pub struct Spec;
+
+#[contractspecfn(name = "Spec", export = false)]
+#[contractclient(name = "DebtTokenClient")]
+pub trait DebtTokenTrait {
+    fn initialize(
+        e: Env,
+        decimal: u32,
+        name: Bytes,
+        symbol: Bytes,
+        pool: Address,
+        underlying_asset: Address,
+    );
+    fn balance(env: Env, id: Address) -> i128;
```

Comment:
Can we make the indents consistent across the codebase? For instance, I see that the `STokenTrait` has a new line between every `fn` declaration.